src/internal_events/azure_blob.rs

```diff
@@ -0,0 +1,52 @@
+use super::InternalEvent;
+use metrics::counter;
+use uuid::Uuid;
+
+#[derive(Debug)]
+pub struct AzureBlobErrorResponse {
+    pub code: hyper::StatusCode,
+}
+
+impl InternalEvent for AzureBlobErrorResponse {
+    fn emit_logs(&self) {
+        error!(message = "HTTP error response.", code = %self.code);
```

Comment:
I think we may want to have a rate limit on this log too just to avoid spamming logs if Azure is down.